### DISCUSSION TITLE
Docker seems completely broken

### DISCUSSION THREAD

**Discussion by androidacy-user** [plausible/analytics#758] [Self-Hosted Support]
Two issues prevent it from working:
1) With the default pass given in docker-compose.yml, postgres auth fails. it should be using the container for postgres, as I don't even have postgresql on the host at all
2) Click house gets a permission denied error for files in /etc/Clickhouse - once again, with default setup for that container.

Are there steps not mentioned at https://plausible.io/docs/self-hosting I have to do?

**Comment by Vigasaurus** [comment]
Do you have log outputs? Those will probably help diagnose the issue. 

Only thing I could recommend at this point is ensuring your user has proper privileges for running docker container (and that your host does too).
